include/cuco/static_multimap.cuh

```diff
@@ -834,6 +860,28 @@ class static_multimap {
       Key const& k,
       KeyEqual key_equal = KeyEqual{}) noexcept;
 
+    /**
+     * @brief Indicates whether the pair `p` exists in the map.
+     *
+     * If the pair `p` was inserted into the map, `contains` returns
+     * true. Otherwise, it returns false. Uses the CUDA Cooperative Groups API to
+     * to leverage multiple threads to perform a single `contains` operation. This provides a
+     * significant boost in throughput compared to the non Cooperative Group
+     * `contains` at moderate to high load factors.
+     *
+     * @tparam PairEqual Binary callable type
+     * @param g The Cooperative Group used to perform the contains operation
+     * @param k The pair to search for
+     * @param pair_equal The binary callable used to compare input pair and slot content
+     * for equality
+     * @return A boolean indicating whether the input pair was inserted in the map
+     */
+    template <typename PairEqual = thrust::equal_to<value_type>>
+    __device__ __forceinline__ bool pair_contains(
+      cooperative_groups::thread_block_tile<ProbeSequence::cg_size> const& g,
+      value_type const& p,
+      PairEqual pair_equal = PairEqual{}) noexcept;
```

Comment:
In addition, we also need to add the version of `pair_contains` here without the `thread_block_tile` parameter.

Reply:
This is exposed to the users, right? So the users don't have that parameter.

Reply:
Unlike single map, multimap needs to handle duplicates. That's the reason that we provide CG APIs only in multimap. Non-CG APIs are provided in single map but are also rarely used. Can you provide more details on your use case i.e. why would non-CG implementation is preferred over the CG ones?

Reply:
>  So the users don't have that parameter.

users can create a CG e.g. https://godbolt.org/z/o6jYYv7Ys